modules/backend/models/EditorSetting.php

```diff
@@ -111,6 +111,13 @@ public function initSettingsData()
         $this->html_paragraph_formats = $this->makeFormatsForTable($this->defaultHtmlParagraphFormats);
     }
 
+    public function afterFetch()
+    {
+        if (!isset($this->value['html_paragraph_formats'])) {
+            $this->html_paragraph_formats = $this->makeFormatsForTable($this->defaultHtmlParagraphFormats);
+        }
+    }
+
```

Comment:
This is a quick fix to initialize the Paragraph Formats table with the default values, if the field doesn't exist in the current editor settings state.

Does it make sense? Am I missing another obvious way of handling default values?